multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -359,12 +312,16 @@ write_files:
                       "-initial-cluster-token",
                       "skydns-etcd"
                     ],
-                    "image": "gcr.io/google_containers/etcd:2.0.9",
+                    "image": "gcr.io/google_containers/etcd:2.2.1",
```

Comment:
Upstream seems to be using `etcd-amd64`: https://github.com/kubernetes/kubernetes/blob/release-1.2/cluster/addons/dns/skydns-rc.yaml.in#L24